app/processing/payment-request/get-completed-payment-request-by-payment-request-id.js

```diff
@@ -6,9 +6,11 @@ const getCompletedPaymentRequestByPaymentRequestId = async (paymentRequestId, tr
     transaction,
     attributes: [
       'paymentRequestId',
```

Comment:
Do not need this value, not used anywhere -- remove here, getPaymentRequest, schema and test files